### DISCUSSION TITLE
What are the goals for Orca Slicer?

### DISCUSSION THREAD

**Discussion by julie777** [OrcaSlicer/OrcaSlicer#639] [Q&A]
The readme lists some of the features, but not specific goals for the project.

I understand why Prusa and Bambu Lab created their slicers to make their printers look as good as possible.
Now Orca is 4 forks away from the original source that I see in the log files. I would guess that none of the slicers can still pull source from the upstream project to stay current with them. Having to fix bugs in multiple variations of a product takes more people and more effort, so I am sure you must have a good reason.

I am not complaining or criticizing. In fact, I am looking for like minded individuals who really want to make the 3D printer ecosystem for hobbyists much better. 

Wouldn't it be great if you had a tool chain that all worked together and made life easier?
There are so many things we could do. I think full automated calibration is a great idea for part of the tool chain.

One thing I would like to see, given the focus is on "Bambu Slicer" is fix it to truly have a LAN only mode. I already have an app that talks to the printer that I have been using for testing, trying to find out why Bambu Slicer has so many issues. Even with a LAN only mode printer it still spends time constantly trying to connect to Bambu Lab servers.

I don't know how large the group working on this project is, or what you plan to focus on. I'll wait for some discussion to see if this is the place for me to get involved.

**Comment by julie777** [comment]
I am still waiting for some discussion on this. 

I have many ideas to improve the 3D printing tool chain. I don't really want to create another slicer fork, so I would like to start a discussion here and see if you @SoftFever are open to other people contributing.

**Reply by robthide37** [reply]
If you are willing to contribute to a slicer I was paying Merril to help me and add features more in line with idex and tool changers etc among other things but he has been swamped so i am now hoping to put it together a few people and i have others that are willing to help with financing it and keeping it open source at same time

**Comment by SoftFever** [comment]
Hi @julie777 I overlooked this post earlier. 
I will find some time to write down my thoughts/purpose/vision on Orca Slicer project later this week.
